src/kernels/exponential.jl

```diff
@@ -35,21 +36,23 @@ kappa(κ::ExponentialKernel, d::Real) = exp(-d)
 iskroncompatible(::ExponentialKernel) = true
 metric(::ExponentialKernel) = Euclidean()
 
-Base.show(io::IO,::ExponentialKernel) = print(io,"Exponential Kernel")
+Base.show(io::IO, ::ExponentialKernel) = print(io, "Exponential Kernel")
 
 ## Alias ##
 const LaplacianKernel = ExponentialKernel
 
 """
-`GammaExponentialKernel([ρ=1.0, [γ=2.0]])`
+    GammaExponentialKernel(; γ = 2.0)
```

Comment:
```suggestion
    GammaExponentialKernel(; γ = 2.0)

```
I assume the default is `2.0` and not `2` due to possible optimization of the hyperparameter?

Reply:
BTW I guess 2 is a standard choice in literature?

Reply:
What do you mean by default is `2.0` and not `2`? I set the default as 2.0 a bit randomly

Reply:
I mean, why using a floating point number instead of an integer as default? My assumption was that you can't really train an integer, and hence the initialization as a (vector of) floating point number might be more suitable.

Reply:
Ah yes, I get it now. \gamma belong to the positive reals. Hence the initialization as a Float